python/pyspark/testing/utils.py

```diff
@@ -22,6 +22,7 @@
 import unittest
 from time import time, sleep
 
+from pyspark.find_spark_home import _find_spark_home
 from pyspark import SparkContext, SparkConf
```

Comment:
```suggestion
from pyspark import SparkContext, SparkConf
from pyspark.find_spark_home import _find_spark_home
```

Reply:
I'm okay with this change, but I'm not sure why we need this change?

Actually, I deliberately put this addtion to the front 😂, because [1]: `imports should be sorted lexicographically`. And flake8 also has a plugin `import-order-style` (not enable in Spark) helps to check it. 

`pysparkfind_spark_home_find_spark_home` should before `pysparkSparkContextSparkConf`.

[1] https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#313-imports-formatting

Reply:
oh okay

Reply:
Actually it says:

> imports should be sorted lexicographically, ignoring case, according to each module’s full package path (the `path` in `from path import ...`)

Since `pyspark.` is after `pyspark` , I think placing `from pyspark import SparkContext, SparkConf` up seems correct :-).

Reply:
oops, I didn't catch the ".", thanks